addons/web/views/base_document_layout_views.xml

```diff
@@ -16,14 +16,14 @@
                             <field name="font" widget="selection" required="1"/>
                             <field name="logo" widget="image" options="{'size': [0, 50]}"/>
                             <label for="primary_color" string="Colors" />
-                            <div class="o_document_layout_colors">
-                                <field name="primary_color" widget="color"/>
-                                <field name="secondary_color" widget="color"/>
-                                <div class="o_custom_colors" title="Reset to logo colors"
+                            <div class="o_document_layout_colors d-flex align-items-end mb-4">
+                                <field name="primary_color" widget="color" class="w-auto m-0 me-1"/>
+                                <field name="secondary_color" widget="color" class="w-auto m-0"/>
+                                <a class="o_custom_colors btn btn-secondary btn-sm position-relative ms-2" role="button" title="Reset to logo colors"
```

Comment:
Do we really need the `position-relative` class ?

Reply:
Yes otherwise the absolute positioned field will take the full width and height of the modal

Reply:
Fair enough